lib/ifquery

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+# Crudely emulate ifquery on systems that don't have it.
+
+import sys
+import glob
+
+def parse_config(interfaces_file, interface):
+    f = open(interfaces_file)
+    intf = ''
+    for line in f:
```

Comment:
Yes, this does work, see: https://docs.python.org/2/tutorial/inputoutput.html#methods-of-file-objects
f.readlines() would also work, but that requires more memory since it loads the whole file into a list before looping over it, as opposed to loading the file one line at a time.